fork-tests/upgrade/1.0.0-1.1.0.ts

```diff
@@ -0,0 +1,132 @@
+import { expect } from "chai";
+import { ethers } from "hardhat";
+import { keccak256, toBeHex, concat } from "ethers";
+import { getStorageAt } from "@nomicfoundation/hardhat-network-helpers";
+import { upgradeFacets } from "../../scripts/upgrade/upgrade-facets";
+import { createClient, fetchExchange } from "@urql/core";
+import fetch from "node-fetch";
+import fs from "fs";
+import path from "path";
+import { readContracts } from "../../scripts/libraries/utils";
+import { prepareTokenFeeBackfillData, prepareOfferBackfillData } from "../../scripts/upgrade-hooks/1.1.0";
+
+const protocolLookupsSlot = "0x769aa294c8d03dc2ae011ff448d15e722e87cfb823b4b4d6339267d1c690d900";
+const offerLookupsSlot = BigInt(protocolLookupsSlot) + 6n;
+const tokenLookupsSlot = BigInt(protocolLookupsSlot) + 7n;
+const env = "test";
+
+function getMappingStorageSlot(mappingSlot: string | bigint, key: string | bigint): string {
+  const paddedKey = toBeHex(BigInt(key), 32);
+  const paddedMappingSlot = toBeHex(BigInt(mappingSlot), 32);
+  const concatenated = concat([paddedKey, paddedMappingSlot]);
+  return keccak256(concatenated);
+}
+
+// Skip by default, but can be run directly with --grep "fork"
+describe("Fork Upgrade from 1.0.1 to 1.1.0", function () {
+  let fermionProtocolAddress: string;
+  let graphQLClient: any;
+
+  beforeEach(async function () {
+    const originalNetwork = await ethers.provider.getNetwork();
+    const originalChainId = originalNetwork.chainId.toString();
+
+    const contractsFile = await readContracts(env);
+    const contracts = contractsFile?.contracts;
+
+    if (!contracts) {
+      throw new Error("No contracts found in contracts file");
+    }
+
+    // Get protocol address and contract instances
+    fermionProtocolAddress = contracts.find((c: any) => c.name === "FermionDiamond")?.address;
+    if (!fermionProtocolAddress) {
+      throw new Error("Protocol address not found");
+    }
+
+    // Setup GraphQL client
+    const subgraphConfig = JSON.parse(
+      fs.readFileSync(path.join(__dirname, "../../scripts/upgrade-hooks/subgraph_config.json"), "utf8"),
+    );
+    const graphQLUrl = subgraphConfig[env][originalChainId].subgraphUrl;
+    graphQLClient = createClient({
+      url: graphQLUrl,
+      exchanges: [fetchExchange],
+      fetch: fetch as any,
+    });
+  });
+
+  context("Backfilling", function () {
+    it("should backfill token fees and offer data during upgrade", async function () {
+      // Get data from subgraph
```

Comment:
I forgot to comment that in the previous review - maybe it's worth disabling the timeout in this test, since backfiling can take quite some time.

```suggestion
      this.timeout(0);
      // Get data from subgraph
```